akka-stream/src/main/protobuf/StreamRefMessages.proto

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+option java_package = "akka.stream";
+option optimize_for = SPEED;
+
+/*************************************************
+  StreamRefs (SourceRef / SinkRef) related formats
+**************************************************/
+
+message SinkRef {
+    required ActorRef targetRef = 1;
```

Comment:
Do we even need that double layer of wrappers? It seems you could just serialize the plain path together with the serialization id.

Reply:
Yeah though wanted that in case of future evolution, like adding an optional to say what kind of ref it is etc..

Reply:
We could still use manifests for evolution but I guess it's fine like this for now.